docs/real-device-config.md

```diff
@@ -222,6 +222,30 @@ This would make the device management more flexible, but you'd need to know abou
 > It is recommended to sign packages with a wildcard (`*`) provisioning profile. Such profiles require a paid Apple Developer account. In case of a free account, you may need to update bundle id properly before building the WebDriverAgent package.
 
 
+### Runs preinstalled WebDriverAgent
+
+`appium:usePreinstalledWDA` capability allows Appium to start a preinstalled WebDriverAgentRunner process without `xcodebuild`.
+The preinstalled WebDriverAgent can be signed with non-paid account.
```

Comment:
I would prefer to have a separate step-by-step tutorial on how to preinstall WDA on the device under test